polaris/utils/misc.py

```diff
@@ -16,3 +19,15 @@ def sluggify(sluggable: SlugCompatibleStringType):
     Converts a string to a slug-compatible string.
     """
     return sluggable.lower().replace("_", "-")
+
+
+def should_verify_checksum(strategy: ChecksumStrategy, dataset: "Dataset") -> bool:
+    """
+    Determines whether a checksum should be verified.
+    """
+    if strategy == "ignore":
+        return False
```

Comment:
Just out of curiosity: What is the scenario where one wants to ignore checksum verification?

Reply:
Good question! It was added because I once - in my infinite wisdom - decided to change how the checksum was computed in the code and I migrated the DB accordingly. This led to a scenario in which any older client could not load any of the datasets or benchmarks anymore, despite the data model being the same...

I will keep it in there for now, mostly as a fail-safe, but maybe I'm being paranoid here!

Reply:
ah makes sense.. backwards compatibility always leads to things like this.